runelite-client/src/main/java/net/runelite/client/chat/ChatMessageManager.java

```diff
@@ -107,7 +107,7 @@ public void onConfigChanged(ConfigChanged event)
 		}
 	}
 
-	@Subscribe
+	@Subscribe(priority = 1) // run after all other onChatMessage plugins
```

Comment:
I don't think this is a good solution for this issue. The League Chat Icons plugin should be modified to not remove color tags instead imo.